pd-client/client.go

```diff
@@ -35,6 +35,8 @@ type Client interface {
 	GetClusterID(ctx context.Context) uint64
 	// GetTS gets a timestamp from PD.
 	GetTS(ctx context.Context) (int64, int64, error)
+	// GetTSAsync gets a timestamp from PD, without block the caller.
+	GetTSAsync(ctx context.Context) *tsoRequest
```

Comment:
This is not a good style(golint seems doesn't complain), but the code actually works.
I just keep the minimal changes, should I rename it to uppercase? @coocood